packages/vuelidate/src/core.js

```diff
@@ -343,13 +345,33 @@ function createMetaFields (results, ...otherResults) {
 
   const $error = computed(() => ($invalid.value && $dirty.value) || false)
 
+  const $touch = () => {
+    // call the root $touch
+    results.$touch()
+    // call all nested level $touch
+    Object.values(allResults.value).forEach((result) => {
```

Comment:
I dont like that we call Object.Values like that, but then again, these two methods are not called often at all...

Reply:
I think that’s alright!